ext/freebsd-getopt/Kconfig

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
```

Comment:
Please use the new license identifier: `LicenseRef-Nordic-5-Clause`

Reply:
Fixed